edb/lib/std/30-datetimefuncs.eql

```diff
@@ -21,21 +21,62 @@
 
 
 CREATE FUNCTION
-std::current_date() -> std::date
+std::datetime_now(mode: std::str='') -> std::datetime
 {
-    FROM SQL 'SELECT current_date';
+    FROM SQL $$
+    SELECT (
+        CASE WHEN "mode" = 'transaction' THEN
+            transaction_timestamp()
+        WHEN "mode" = 'statement' THEN
+            statement_timestamp()
+        ELSE
```

Comment:
I don't like that this function can accept any garbage.  I also think that having more than 2 different modes of operation is way too much for a single function. Let's discuss splitting this into three functions.

Reply:
`std::datetime_of_transaction`, `std::datetime_of_statement`, `std::datetime_now` seem like reasonable names to me.

Reply:
I've updated the PR.